watchgod/main.py

```diff
@@ -74,11 +70,9 @@ def __init__(
         debounce: int = 1600,
         normal_sleep: int = 400,
         min_sleep: int = 50,
-        stop_event: Optional[asyncio.Event] = None,
-        loop: Optional[asyncio.AbstractEventLoop] = None,
+        stop_event: Optional[anyio.Event] = None,
```

Comment:
Or is it? Maybe this should actually accept `Union[anyio.Event, asyncio.Event, trio.Event, None]`

Reply:
Proposed in #99.